### PR TITLE
Update Live with current Master

### PR DESCRIPTION
**Pull request by rpetrusha**

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -1125,6 +1125,30 @@
...
+            "source_path": "docs/fsharp/tutorials/type-providers/accessing-an-odata-service.md",
+            "redirect_url": "docs/fsharp/tutorials/type-providers/index"
+        },
+        {
+            "source_path": "docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-dbml.md",
+            "redirect_url": "docs/fsharp/tutorials/type-providers/index"
+        },
+        {
+            "source_path": "docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx.md",
+            "redirect_url": "docs/fsharp/tutorials/type-providers/index"
```
Comment (pkulikov) [inline .openpublishing.redirection.json]: @cartermp `redirect_url` are not correct: *docs* substring should be replaced with *dotnet*

Comment (mairaw) [reply .openpublishing.redirection.json]: Good catch @pkulikov. I'll make a new PR to fix that.

**Comment by pkulikov** [comment]
@mairaw my commit breaks the build here, but not when it was still a PR. Is it because samples master should be merged to live as well?

**Comment by mairaw** [comment]
Yes, I just did that @pkulikov. We're still learning the new workflow. 😄

**Pull request merged**